### PR TITLE
Active admin

### PR DESCRIPTION
**Pull request by alanlamaswolox**
## Summary
Active admin installed and backoffice created

## Trello
https://trello.com/c/VWYMLlWi/10-20-instalar-y-configurar-active-admin

### CODE REVIEW

File: db/migrate/20181001163831_create_active_admin_comments.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class CreateActiveAdminComments < ActiveRecord::Migration::Current
```
Comment (fedeest) [inline db/migrate/20181001163831_create_active_admin_comments.rb]: Wait, what is this?

Comment (alanlamaswolox) [reply db/migrate/20181001163831_create_active_admin_comments.rb]: I something that the gem created automatically when i run "rails g active_admin:install"

Comment (alanlamaswolox) [reply db/migrate/20181001163831_create_active_admin_comments.rb]: *ran

Comment (nicozar95) [inline db/migrate/20181001163831_create_active_admin_comments.rb]: Thougth this is okay if you want to have active admin coments, its not required in our app to have those

Comment (nicozar95) [reply db/migrate/20181001163831_create_active_admin_comments.rb]: Soo. please remove this migration

**Review by nicozar95 — CHANGES_REQUESTED** [review]
Remove the unecesarry migration please

**Review by fedeest — APPROVED** [review]

**Review by nicozar95 — APPROVED** [review]

**Pull request merged**